pkg_resources/tests/test_pkg_resources.py

```diff
@@ -190,6 +191,83 @@ def test_setuptools_not_imported(self):
         subprocess.check_call(cmd)
 
 
+def touch_file(path):
```

Comment:
Can you add a `TODO` comment about dropping this when Python 2 support is dropped? As of Python 3.4 `pathlib.Path` objects have a `touch` method.